### PR TITLE
Aerogear 7687 - fix images and issues 

### PR DESCRIPTION
**Pull request by finp**
preview at https://mobile-security.netlify.com/mobile-security/latest/#aerogear-mobile-security

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+runtime:
+  cache_dir: ./.cache/antora
+site:
+  title: AeroGear Mobile Security
+  url: https://www.aerogear.org
+  start_page: mobile-security::index
+content:
+  sources:
+  - url: https://github.com/finp/mobile-security
```
Comment (laurafitzgerald) [inline site.yml]: Is this file necessary for the shipped version or is it for building locally?

Comment (finp) [reply site.yml]: will remove
